.github/workflows/release.yml

```diff
@@ -137,6 +137,8 @@ jobs:
       - name: Get Tag From Environment
         id: get-tag
         run: printf '::set-output name=tag::%s' "$(printf '%s' "${{ github.ref }}" | sed 's/refs\/tags\///')"
+      - name: Build Binaries
+        run: make release-test
```

Comment:
I believe this needs the GITHUB_TOKEN environment variable set? I can't remember if that was only failing because the repo was private, or if it's still an issue even with the repo being public.
```yaml
env:
  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
Oh, this is doing a snapshot, that's why it doesn't error out (don't need GITHUB_TOKEN for snapshot builds).

The internal version for the collector won't be right w/ the snapshot builds, we should avoid the snapshot flag if possible here.

Reply:
I wanted to ask this also this runs the entire release process do we just want the binaries or do we need something special that comes with doing the entire release? If it's just the binaries we could use `make build-all` or [goreleaser build](https://goreleaser.com/cmd/goreleaser_build/).

Reply:
Goreleaser `build` will only build for the goos/goarch of the runner. We could use `build-all`. We want to avoid a snapshot build, so `release-test` was the wrong choice.

The goal is to use the same binaries shipped with packages. I might experiment with copying the binaries from the build (linux, darwin, windows) stages, and avoid building new binaries all together. That seems like the best solution, maybe somewhat complex.

Reply:
Goreleaser build will build based on the environment variables GOOS and GOARCH - you can set these for the job to build for a specific arch + platform, if you desire.